### PR TITLE
Add product notifier to UI for copying attributes

### PR DESCRIPTION
**Pull request by mc12345678**
Add a notifier to support display of additional options when copying a product.

### CODE REVIEW

File: admin/product.php [hunk 1]
```diff
@@ -232,6 +232,7 @@ function init()
           $contents[] = array('text' => '<br />' . TEXT_COPY_ATTRIBUTES . '<br />' . zen_draw_radio_field('copy_attributes', 'copy_attributes_yes', true) . ' ' . TEXT_COPY_ATTRIBUTES_YES . '<br />' . zen_draw_radio_field('copy_attributes', 'copy_attributes_no') . ' ' . TEXT_COPY_ATTRIBUTES_NO);
 // future          $contents[] = array('align' => 'center', 'text' => '<br />' . ATTRIBUTES_NAMES_HELPER . '<br />' . zen_draw_separator('pixel_trans.gif', '1', '10'));
           $contents[] = array('text' => '<br />' . zen_image(DIR_WS_IMAGES . 'pixel_black.gif','','100%','3'));
+          $zco_notifier->notify('NOTIFY_ADMIN_PRODUCT_COPY_TO_ATTRIBUTES', $contents, $contents);
```
[the remark this answers is not in the document]
Comment (mc12345678) [reply admin/product.php]: Technically neither is "needed" as this code is neither within a class nor a function allowing `$contents` to be in the global space and retrievable from that.  That said, in the adoption for ZC versions pre-1.5.3, it at least offers the data to be seen/used.  If it were to be `empty` I would be just as fine with empty quotes `''`, an empty array `array()` or just plain NULL for the first parameter.

[the remark this answers is not in the document]
Comment (drbyte) [reply admin/product.php]: v1.5.3 was released in 2014. Officially no longer supported. Time to stop coding for such old dependencies.

[the remark this answers is not in the document]
Comment (mc12345678) [reply admin/product.php]: As said, technically no parameter is needed; however, thinking towards some sort of future, having an editable parameter supports maintaining the same notifier in the same code section to be able to be used without further code modification.  So on a serious note, what would be the preferred first parameter? In order to use a second parameter, the first must have something.  Is there any reason for it to specifically be an empty `array()` versus some other "empty" data element?

[the remark this answers is not in the document]
Comment (drbyte) [reply admin/product.php]: I prefer some empty element. Content with null if it doesn't trigger any errors.

[the remark this answers is not in the document]
Comment (drbyte) [reply admin/product.php]: Upon further review, I think `$pInfo` adds some context that might be useful to the observer. Updated.

**Comment by drbyte** [comment]
Thanks!

**Pull request merged**